### PR TITLE
Fix several misc bugs

### PR DESCRIPTION
**Pull request by AndrewLester**
## Status:

:rocket: Ready

## Description

Fix several misc bugs...

### CODE REVIEW

File: pages/api/[...mockApiRoute].ts [hunk 1]
```diff
@@ -97,7 +102,7 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
         new URLSearchParams({ linkSuccess: linkSuccess.toString() });
       res.redirect(307, redirectURL);
       return;
-    } else if ('enode/clear-users') {
+    } else if (mockApiRoute === 'enode/clear-users') {
```
Comment (AndrewLester) [inline pages/api/[...mockApiRoute].ts:105]: This took me a while to realize...

Comment (ericcccsliu) [reply pages/api/[...mockApiRoute].ts:105]: lololol

File: pages/api/[...mockApiRoute].ts [hunk 2]
```diff
@@ -106,4 +111,21 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
...
+    const date = new Date(parseNowcastingDatetime(value.target_datetime_utc));
+    addMilliseconds(date, date.getTime() - first.getTime());
+    return {
+      ...value,
+      target_datetime_utc: date.toISOString(),
+    };
+  });
+  return forecastData;
+}
+
```
Comment (ericcccsliu) [inline pages/api/[...mockApiRoute].ts:130]: great addition

File: pages/api/[...mockApiRoute].ts [hunk 3]
```diff
@@ -54,12 +57,14 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
...
   if (req.method == 'GET') {
     if (
-      mockApiRoute === 'sites/b97f68cd-50e0-49bb-a850-108d4a9f7b7e/pv_actual'
+      mockApiRoute === 'sites/725a8670-d012-474d-b901-1179f43e7182/pv_actual'
     ) {
+      fakeDates(pvActualJson as any);
       res.status(200).json(pvActualJson);
     } else if (
-      mockApiRoute === 'sites/b97f68cd-50e0-49bb-a850-108d4a9f7b7e/pv_forecast'
+      mockApiRoute === 'sites/725a8670-d012-474d-b901-1179f43e7182/pv_forecast'
```
Comment (ericcccsliu) [inline pages/api/[...mockApiRoute].ts:65]: oh yay now we won't have to change around our siteUUID between testing and deployment

File: components/SiteCard.tsx [hunk 4]
```diff
@@ -1,69 +1,75 @@
...
 interface SiteCardProps {
   href?: string;
+  onClick?: () => void;
   siteUUID: string;
 }
 
-const SiteCard: FC<SiteCardProps> = ({ href, siteUUID }) => {
-  const { forecastData, client_site_name, installed_capacity_kw, isLoading } =
-    useSiteData(siteUUID);
+// eslint-disable-next-line react/display-name
```
Comment (ericcccsliu) [inline components/SiteCard.tsx]: why is this line necessary?

**Review by ericcccsliu — APPROVED** [review]

File: components/SiteCard.tsx (hunk 4 above)
Comment (AndrewLester) [reply components/SiteCard.tsx]: Fixed by setting `displayName` on the component.
